internal/qutil/util.go

```diff
@@ -0,0 +1,15 @@
+package qutil
```

Comment:
Suggest to name this package `rand` or `random`. Then we can use `random.String()` instead of `qutil.RandomString()`

Reply:
Renamed to rand.